.github/workflows/youtube_base.yml

```diff
@@ -13,6 +13,8 @@ jobs:
         python-version: ["3.10"]
         poetry-version: ["1.7.1"]
     runs-on: ubuntu-latest
+    env:
+      API_KEY: ${{ secrets.API_KEY }}
```

Comment:
What is meant by "secrets"?

How the value is passed while the yml file analyzation?

Reply:
The secret "API_KEY" is a variable in GitHub actions env.
First I added the variable to the repository env and after I can add this variable to the yml script.
https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions